### PR TITLE
Accessibility bugs

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/pages/common/section/components/fields/bulletpoint-list.njk [hunk 1]
```diff
@@ -43,6 +43,10 @@
             </div>
           {% else %}
             <div data-test-id="field-{{ field.id }}">
+              {{ label({
+                "text": params.question.id + ' ' + loop.index,
+                "classes": "nhsuk-u-visually-hidden"
+              }) }}
```
Comment (alibombaye) [inline app/pages/common/section/components/fields/bulletpoint-list.njk]: Can we update the unit tests to check for this.

File: app/pages/common/section/components/fields/bulletpoint-list.test.js [hunk 2]
```diff
@@ -180,4 +180,36 @@ describe('bulletpoint-list', () => {
       expect($('[data-test-id="bulletpoint-list-additional-advice"]').length).toEqual(0);
     });
   }));
+
+  it('should not render the accessibility label for listing inputs', createTestHarness(setup, (harness) => {
```
Comment (alibombaye) [inline app/pages/common/section/components/fields/bulletpoint-list.test.js]: should the `not` be there?

Comment (BenBJSS) [reply app/pages/common/section/components/fields/bulletpoint-list.test.js]: yeah you're right, removed it

File: app/pages/common/section/components/fields/bulletpoint-list.test.js [hunk 3]
```diff
@@ -180,4 +180,36 @@ describe('bulletpoint-list', () => {
...
+              data: '',
+            },
+          ],
+        },
+      },
+    };
+
+    harness.request(context, ($) => {
+      expect($('[data-test-id="bulletpoint-list-fields"] label.nhsuk-u-visually-hidden').length).toEqual(3);
+      expect($('[data-test-id="bulletpoint-list-fields"] label.nhsuk-u-visually-hidden')[0].children[0].data.trim()).toEqual('field 1');
```
Comment (alibombaye) [inline app/pages/common/section/components/fields/bulletpoint-list.test.js]: can we remove the `children[0]` and have a more specific selector?

Comment (BenBJSS) [reply app/pages/common/section/components/fields/bulletpoint-list.test.js]: I couldn't find another way to do it as it's using a nhs label component

**Pull request merged**